public/app/features/dashboard/dashgrid/PanelChrome.tsx

```diff
@@ -152,7 +152,6 @@ export class PanelChrome extends PureComponent<Props, State> {
         timeRange: timeData.timeRange,
         timeInfo: timeData.timeInfo,
         widthPixels: width,
-        minInterval: undefined, // Currently not passed in DataPanel?
```

Comment:
ah, that is a bug this should be passed from panel.interval prop , (similar to how panel.maxDataPoints is passed)